skops/io/_cli.py

```diff
@@ -0,0 +1,113 @@
+from __future__ import annotations
+
+import argparse
+import logging
+import os
+import pathlib
+import pickle as pkl
+from typing import Optional
+
+from skops.io import dumps, get_untrusted_types
+
+
+def _convert(
+    input_file: os.PathLike, output_dir: pathlib.Path, is_trusted: bool = False
+):
+    model_name = pathlib.Path(input_file).stem
+
+    logging.info(f"Converting {model_name}")
+
+    with open(input_file, "rb") as f:
+        obj = pkl.load(f)
+    skops_dump = dumps(obj)
+
+    untrusted_types = get_untrusted_types(data=skops_dump)
+
+    if not untrusted_types:
+        logging.debug(f"No unsafe types found in {model_name}.")
+    else:
+        untrusted_str = "\n".join(untrusted_types)
+
+        logging.warning(
+            "Unsafe Types Detected!\n"
+            f"While converting {model_name}, "
+            "the following unsafe types were found: \n"
+            f"{untrusted_str}\n"
+        )
+
+        if not is_trusted:
+            logging.warning(
+                f"Model {model_name} will not be converted due to unsafe types.\n"
+                "To convert this anyway, add `-t` to this command."
+            )
+            return
```

Comment:
I think we should always convert. The unsafe part is loading a pickle file, and we've already done it at this point. So there's no point in not converting. 

Instead, we can print a `skops.io.load` command which users need to use (with `trusted` arg) to successfully load a converted file.

Reply:
That makes sense to me. Would that mean removing the `--trusted` flag entirely, and just printing a statement that unknown types were found, and `--trusted` would need to be passed in during load?

Reply:
yes!